vulture/core.py

```diff
@@ -29,6 +29,7 @@
 import ast
 import codecs
 from fnmatch import fnmatchcase
+from lines import estimate_lines
```

Comment:
local imports should be separated from stdlib imports by an empty line. Maybe it would be good to test these things with isort.